### ISSUE TITLE
Chitchat Project Broken 

### ISSUE THREAD

**Issue by engahmed1190** [botfront/botfront#808]
⚠️ Before posting a new issue, please make sure that your issue hasn't already been resolved in another issue.

**Describe the bug**
By Default Botfront opens in chitchat project. This error appears on the chitchat dialog page` /project/chitchat-YFju788Pq/dialogue`

Any way I can force it to open over the created project. also, why this error appears

**To Reproduce**
1. Go to 'http://SERVERIP/project/chitchat-YFju788Pq/dialogue'
2. See error

`
TypeError: Cannot read property 'length' of undefined
    at eval (/imports/ui/components/stories/StoryGroupTree.jsx:1)
    at ja (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at ss (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at cs (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at fs (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at Js (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at Xs (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at Vs (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at vs (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at Mo (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:49)
    at 4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:1849
    at 4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:1849
    at t._compute (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:1815)
    at t._recompute (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:1815)
    at Object.o._runFlush (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:1815)
    at a (4af93d4b12a5dcc1d1ba3a4fab165bd8d0ef43b7.js?meteor_js_resource=true:1)
`

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by MatthieuJnon** [comment]
Does this happen on a new project you just created ? What happens if you go on `http://SERVERIP/project/bf/dialogue` ?

**Comment by engahmed1190** [comment]
No, it works fine with the created project, trained successfully

**Comment by engahmed1190** [comment]
Hello @MatthieuJnon . What do you think cause of this error
